packages/website/styles/layouts/userRequestsModals.scss

```diff
@@ -52,18 +52,18 @@
   > * {
     position: relative;
   }
-}
 
-.user-request-modal__heading {
-  font-size: 1.5rem;
-}
+  .user-request-modal__heading {
```

Comment:
I also see the darker green here in github? Might not be worth worrying about but just fyi.

Reply:
I moved the `}` from like 55 in the old diff down

Reply:
The problem I mentioned yesterday turned out to not be an issue with kwes-forms, but with the fact that we hadn't scoped our styles to the component instance and so the styling was bleeding across all of them

Reply:
@drewdelano good catch :)